RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -880,9 +912,9 @@ class FeedbackBoardContainer extends React.Component<FeedbackBoardContainerProps
     this.setState({ isRetroSummaryDialogHidden: true });
   }
 
-  private updateBoardMetadata = async (title: string, maxvotesPerUser: number, columns: IFeedbackColumn[]) => {
+  private updateBoardMetadata = async (title: string, maxvotesPerUser: number, columns: IFeedbackColumn[], isIncludeTeamEffectivenessMeasurement: boolean, isBoardAnonymous: boolean, shouldShowFeedbackAfterCollect: boolean, displayPrimeDirective: boolean, permissions: IFeedbackBoardDocumentPermissions) => {
```

Comment:
Minor thing to consider for human readability - break up some of the longer lines.

Reply:
here and elsewhere in the PR. IIRC, there's currently no hard requirement, but I personally shoot for ~100 characters per line.

Reply:
I'm ok with longer lines, since there are few, and monitors getting wider everyday :)

It's good to be conscious about it, because it's a code smell. There could be a reason to split the long line by introducing variables.

In this case, I don't think having a named method would make much difference.

We can evaluate each case separately 👍

Reply:
Totally agree, the amount of parameters added was due to the way to passthrough the `update` call, and would love to refactor this in isolation.